src/ldp/http/ResponseWriter.ts

```diff
@@ -7,4 +7,4 @@ import type { ResponseDescription } from './response/ResponseDescription';
  * Response depends on the operation result and potentially which errors was thrown.
```

Comment:
This comment still true?

Reply:
Sort of, since the ResponseDescription it takes as input can also come from an error that was thrown.